src/Sulu/Bundle/AdminBundle/Resources/js/components/Block/tests/Block.test.js

```diff
@@ -120,9 +126,10 @@ test('Changing the type should call the onTypeChange callback', () => {
         type1: 'Type 1',
         type2: 'Type 2',
     };
-    const block = shallow(
+
+    render(
         <Block
-            activeType="type1"
+            activeType={Object.keys(types)[0]}
```

Comment:
Would keep here `"type1"` tests are fine to be repetitive that make them better readable and understandable. Else you need always first check okay what is inside `types` to know what here is given.